frontend/express/public/core/user-management/templates/drawer.html

```diff
@@ -0,0 +1,173 @@
+                </div>
+                <div class="input" v-if="drawerScope.editedObject.permission">
+                  <el-select placement="bottom-start" @change="onAdminAppsChanged" class="user-management-drawer-content__app-selector" v-model="drawerScope.editedObject.permission._.a" multiple placeholder="Select apps">
+                    <el-option
+                      v-for="item in apps"
+                      :key="item.value"
+                      :label="item.label"
+                      :value="item.value">
+                    </el-option>
+                  </el-select>
+                </div>
+              </div>
+            </div>
+            <div v-if="!drawerScope.editedObject.global_admin" :key="index" v-for="(set, index) in permissionSets" class="user-management-drawer-content__drawer-sub-section">
+              <div class="user-management-drawer-content__section-title">
+                <div>
+                  <span class="user-management-drawer-content__section-header">GRANT USER ACCESS TO APP(S) <cly-tooltip-icon icon="ion ion-help-circled"></cly-tooltip-icon></span>
+                </div>
+                <div>
+                  <cly-more-options size="small" @command="handleCommand($event, index)">
+                    <el-dropdown-item command="remove-set">Remove permission set</el-dropdown-item>
+                  </cly-more-options>
+                </div>
+              </div>
+              <div class="user-management-drawer-content__section-content">
+                <div class="user-management-drawer-content__label user-management-drawer-content__label--pl-10">
+                  <div class="text-small text-heading user-management-drawer-content__label--text-secondary">Some app selection related description can be placed here</div>
+                </div>
+                <div class="user-management-drawer-content__input">
+                  <el-select placement="bottom-start" @change="onUserAppsChanged(index)" class="user-management-drawer-content__app-selector" v-model="drawerScope.editedObject.permission._.u[index]"  multiple placeholder="Select apps">
+                    <el-option
+                      v-for="item in apps"
+                      :key="item.value"
+                      :label="item.label"
+                      :value="item.value">
+                    </el-option>
+                  </el-select>
+                </div>
+                <div class="user-management-drawer-content__permission-table-header">
+                  <div class="user-management-drawer-content__feature-name-col">
+                    FEATURE
+                  </div>
+                  <div class="user-management-drawer-content__permission-type-cols">
+                    <div class="user-management-drawer-content__permission-type">
+                      <cly-check skin="tick" @input="setPermissionByType(index, 'c')" v-model="permissionSets[index].c.all" v-bind:label="'CREATE'"></cly-check>
```

Comment:
Same cly-check and string literal warning.

Reply:
Also you use @input and v-model together, which is kind of odd. 

```
@input="setPermissionByType(index, 'c')" 
v-model="permissionSets[index].c.all"
```

So this `<input v-model="field">` is basically shorthand for the following:

```
<input
   :value="field"
   @input="field = $event.target.value"
>
```
Any particular reason for this?